docs/core_docs/docs/modules/model_io/quick_start.mdx

```diff
@@ -0,0 +1,239 @@
+
+<Tabs>
+  <TabItem value="openai" label="OpenAI" default>
+
+First we'll need to install the LangChain OpenAI integration package:
+
+:::tip
+See [this section for general instructions on installing integration packages](/docs/get_started/installation#installing-integration-packages).
+:::
+
+```bash npm2yarn
+npm install @langchain/openai
+```
+
+Accessing the API requires an API key, which you can get by creating an account and heading [here](https://platform.openai.com/account/api-keys). Once we have a key we'll want to set it as an environment variable by running:
+
+```shell
+export OPENAI_API_KEY="..."
+```
+
+We can then initialize the model:
+
+```typescript
+import { OpenAI, ChatOpenAI } from "@langchain/openai";
+
+const llm = new OpenAI({
+  modelName: "gpt-3.5-turbo-instruct",
+});
+const chatModel = new ChatOpenAI({
+  modelName: "gpt-3.5-turbo",
+});
+```
+
+If you can't or would prefer not to set an environment variable, you can pass the key in directly via the `openAIApiKey` named parameter when initiating the OpenAI LLM class:
+
+```typescript
+const model = new ChatOpenAI({
+  openAIApiKey: "<your key here>",
+});
+```
+
+  </TabItem>
+  <TabItem value="local" label="Local">
+
+[Ollama](https://ollama.ai/) allows you to run open-source large language models, such as Llama 2 and Mistral, locally.
+
+First, follow [these instructions](https://github.com/jmorganca/ollama) to set up and run a local Ollama instance:
+
+- [Download](https://ollama.ai/download)
+- Fetch a model via e.g. `ollama pull mistral`
+
+Then, make sure the Ollama server is running. Next, you'll need to install the LangChain community package:
+
+:::tip
+See [this section for general instructions on installing integration packages](/docs/get_started/installation#installing-integration-packages).
+:::
+
+```bash npm2yarn
+npm install @langchain/community
+```
+
+And then you can do:
+
+```typescript
+import { Ollama } from "@langchain/community/llms/ollama";
+from { ChatOllama } from "@langchain/community/chat_models/ollama";
+
+const llm = new Ollama({
+  baseUrl: "http://localhost:11434", // Default value
+  model: "mistral",
+});
+const chatModel = new ChatOllama({
+  baseUrl: "http://localhost:11434", // Default value
+  model: "mistral",
+});
+```
+
+  </TabItem>
+</Tabs>
+
+Both `llm` and `chatModel` are objects that represent configuration for a particular model.
+You can initialize them with parameters like `temperature` and others, and pass them around.
+The main difference between them is their input and output schemas.
+
+- The LLM class takes a string as input and outputs a string.
+- The ChatModel class takes a list of messages as input and outputs a message.
+
+For a deeper conceptual explanation of this difference please see [this documentation](/docs/modules/model_io/concepts)
+
+We can see the difference between an LLM and a ChatModel when we invoke it.
+
+```ts
+import { HumanMessage } from "@langchain/core/messages";
```

Comment:
```suggestion
We can see the difference between an LLM and a ChatModel when we invoke it.

```bash npm2yarn
npm install @langchain/core
```

```ts
import { HumanMessage } from "@langchain/core/messages";
```

Reply:
Don't commit the above^ looks like github screwed up the code block formatting

Reply:
This comment is just proposing add a block showing how to install core since no mention of that occurs before this

Reply:
I think that one will 100% be installed with `langchain` in the future so I think it's ok, unlike `@langchain/openai` and community